python/xorbits/deploy/kubernetes/config.py

```diff
@@ -623,8 +630,14 @@ def build_template_spec(self) -> Dict:
         result = {
             "containers": [self.build_container()],
             "initContainers": self.config_init_containers(),
-            "volumes": [vol.build() for vol in self._volumes],
         }
+
+        if self._external_storage != "juicefs":
+            result["volumes"] = [vol.build() for vol in self._volumes]
+        else:
+            result["volumes"] = [
+                {"name": "data", "persistentVolumeClaim": {"claimName": "juicefs-pvc"}}
+            ]
```

Comment:
If the user specifies both the directory they want to mount and the juicefs, is there a problem here?